### ISSUE TITLE
[PostgreSQL] allow database url to begin by `posgresql://`

### ISSUE THREAD

**Issue by ipfaze** [Scalingo/cli#934]
Currently the only acceptable url must begin with `postgres://` otherwise the addons will not be detected.
Intercom: https://app.intercom.com/a/inbox/w4oogu7s/inbox/shared/all/conversation/12375700160246?view=List

**Comment by EtienneM** [comment]
I don't understand this ticket. Can you explain which command you try to execute, what error message you see, and what you expect to see?

**Comment by ipfaze** [comment]
According to the Postgres documentation about [connection string](https://www.postgresql.org/docs/current/libpq-connect.html#LIBPQ-CONNSTRING).
>The URI scheme designator can be either `postgresql://` or `postgres://`.

As you can see below, we actually only take in charge the `postgres://` one.
https://github.com/Scalingo/cli/blob/90f1a99f1f1270f020ec1110e5f88ae384249c07/db/pgsql_console.go#L21

And for some reason, some customers has their connection string beginning by `postgresql://`.

So when trying to connect using: `scalingo -a myapp pgsql-console`.
I expect to get a pgsql-console but I got the error: 
```
!     An error occurred:
       no scalingo_postgresql addon detected
```

**Comment by EtienneM** [comment]
So if I understand correctly, this ticket is about the `pgsql-console` command? When the content of the environment variable `SCALINGO_POSTGRESQL` starts with `postgresql://` rather than `postgres://`?

**Comment by ipfaze** [comment]
Exactly, it's related to `pgsql-console` when the environment variable `SCALINGO_POSTGRESQL` starts with `postgresql://` rather than `postgres://`.
